docs/glossary.md

```diff
@@ -0,0 +1,87 @@
+# Glossary
+
+### Attribute
+A characteristic of a material, process, or measurement. Attributes can be a GEMD property, condition, or parameter.
+
+### Attribute Template
+Attribute templates define a domain concept (e.g. density, yield strength), and include bounds to define acceptable values for a given attribute.
```

Comment:
I feel like the glossary needs to be more conceptual than structural (e.g. these objects link together in these specific ways), but this doc is a little half and half at the moment. Thoughts?

Reply:
All of these terms should already exist within the broader document, and all of the linking is defined in that.  I'd think this should be super succinct and have linking into the deeper descriptions.  Which I think is saying I agree with you.

Reply:
For time-boxing sake, what would you think about the following acceptance criteria:
 pass 1: glossary without examples or hrefs to the docs
 pass 2: glossary with consistent contextual examples (call out boxes? something nice?)
 pass 3: hrefs to docs: I'm a bit  hesitant to do this because of the maintenance overhead.

Reply:
Also I asked stable diffusion to make me an image of "time-boxing":
![image](https://user-images.githubusercontent.com/117872475/218852117-e6398f06-d776-4ead-8a9a-6755a3737392.png)